packages/snaps-utils/src/snaps.ts

```diff
@@ -306,9 +272,6 @@ export const HttpSnapIdStruct = intersection([
 
 export const SnapIdStruct = union([NpmSnapIdStruct, LocalSnapIdStruct]);
 
-export type ValidatedSnapId = Opaque<string, typeof snapIdSymbol>;
```

Comment:
Looks like this has been removed entirely? Are we sure about this? It seemed like a decent idea to distinguish between validated and non-validated IDs

Reply:
This was moved to `snaps-sdk` (as `SnapId`), since it's required for the request types. I removed the `Validated` prefix, since that doesn't make much sense to users. Non-validated IDs are just strings, so I've changed it to reflect that.